pom.xml

```diff
@@ -135,8 +135,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.3.2</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>1.8</source>
+					<target>1.8</target>
 				</configuration>
```

Comment:
Source and target can be changed as below. in future changes of java version one place to go to change
```
**Set**
<properties>
           <java.version>1.8</java.version>
<properties>

**Access it as**
<source>${java.version}</source>
<target>${java.version}</target>

```